python/sglang/srt/model_config.py

```diff
@@ -20,8 +20,11 @@ def __init__(
         # Unify the config keys for hf_config
         self.context_len = get_context_length(self.hf_config)
         self.head_dim = self.hf_config.hidden_size // self.hf_config.num_attention_heads
-        self.num_key_value_heads = self.hf_config.num_key_value_heads
         self.num_attention_heads = self.hf_config.num_attention_heads
+        try:
+            self.num_key_value_heads = self.hf_config.num_key_value_heads
+        except Exception as e:
```

Comment:
do not use a general exception. Can you try something like `KeyNotFoundError`

Reply:
I have change it by using `getattr` method